cfme/utils/browser.py

```diff
@@ -136,6 +136,9 @@ def _firefox_profile(self):
         return _load_firefox_profile()
 
     def processed_browser_args(self):
+        if 'firefox_profile' in self.browser_kwargs:
```

Comment:
**REQUIRED** since that happens lets remove the cache, simplify `__init__` and put

```
if self.webdriver_class is webdriver.Firefox:
    self.browser_kwargs['firefox_profile'] = _load_firefox_profile()
``` here

Reply:
whops, my proposal is not cleanly handling a remote firefox webdriver